exts/cesium.omniverse/cesium/omniverse/ui/attributes/tileset_attributes_widget.py

```diff
@@ -3,12 +3,12 @@
 from omni.kit.property.usd.usd_property_widget import SchemaPropertiesWidget
 import omni.ui as ui
 from ...bindings import ICesiumOmniverseInterface
-from cesium.usd.plugins.CesiumUsdSchemas import TilesetAPI as CesiumTilesetAPI
+from cesium.usd.plugins.CesiumUsdSchemas import Tileset as CesiumTilesetPrim
```

Comment:
The `Prim` suffix can be removed - we don't include it in the Python code

```suggestion
from cesium.usd.plugins.CesiumUsdSchemas import Tileset as CesiumTileset
```